src/MapViewController.swift

```diff
@@ -76,7 +76,8 @@ open class MapViewController: UIViewController, LocationManagerDelegate {
   var currentRouteMarker: TGMapMarkerId?
   open var shouldFollowCurrentLocation = false
   open var findMeButton = UIButton(type: .custom)
-  open var currentAnnotations: [PeliasMapkitAnnotation : TGMapMarkerId] = Dictionary()
+  open var annotationToMarkerId: [PeliasMapkitAnnotation : TGMapMarkerId] = Dictionary()
```

Comment:
Agreed on the initial problem mentioned; though the pure swift solution is more memory efficient but lookups in one way would get expensive.

What about a `MapViewMarker` holding both marker ids? Those seems to always be coupled if I understand this correctly, it would then be possible to have single storage in a dictionary.